components/ProductCard.js

```diff
@@ -0,0 +1,117 @@
+import styled from 'styled-components';
+import QuantitySelector from './QuantitySelector';
+import { useState } from 'react';
+
+export default function ProductCard({ product }) {
+  const [showDetails, setShowDetails] = useState();
+
+  return (
+    <>
+      <StyledCard key={product.id}>
+        <StyledImage src={product.image} alt={product.name} />
```

Comment:
```suggestion
        <StyledImage src={product.image} alt="" />
```
in line 13 the name is already available for screen readers

Reply:
as far as I know alt names also have SEO relevance. I would keep it.